docs/release-notes/saros-i_0.1.0.md

```diff
@@ -0,0 +1,234 @@
+
+<details>
+
+With the current workaround introduced in [PR #417](https://github.com/saros-project/saros/pull/417), Saros/I always holds a reference to a specific IntelliJ project.
+Only modules of this project can be shared.
+
+The held project is determined on runtime as follows:
+- Initially, the first opened IntellJ project is used.
+- Whenever a new project is opened, Saros checks if the held project object is still valid (if the project represented by the object is still open).
+If the object is no longer valid, it will be replaced with the newly opened project.
+As a result, you can now only share modules of the newly opened project.
+
+If you only have one project open at a time or always open the project you want to share modules of first, this restriction should not be a noticeable.
+
+If you want to change the shareable project, you will have to close the currently shareable project and then open the project you want to share modules of.
+If you have a hard time figuring out which project is currently selected as shareable, you can resort to closing all open projects before opening the project to share.
+
+The functionality described above still leaves the possibility of entering a headless state:
+If you close the currently shareable project and then don't open a new project, Saros is left in a headless state where it does not have a valid reference to a project.
+Trying to start a session in this state will lead to exceptions. This headless state can be resolved by opening a new project.
+
+</details>
+
+#### Module Restrictions
+
+You can currently only share a single module.
+Sharing a module will only share resources belonging to that module, not resources belonging to sub-module located inside a content root of the module.
+
+Furthermore, a module has to adhere to the following restrictions to be shareable through:
+
+- The module must have exactly one content root.
+- The module must be located under the project content root.
+- The module file of the module must be located in the base directory of the content root.
+- The module must not be a project module.
+- The module file for the module must not contain absolute paths (or paths in general that can't be resolved by other participants, like network drives).
+
+#### Number of Participants
+
+Currently, Saros/I is restricted to two-participant sessions, meaning you can only create session containing the host and a single client.
+
+#### File Deletion Restrictions
+
+As mentioned in the section [Module Restrictions](#module-restrictions), Saros/I does not include sub-modules when sharing a module.
+As a consequence, such sub-modules might not be present for all session participants.
+If a participant deletes a shared directory that contains a sub-module in the local setup of another participant, this sub-module will be deleted without any notice.
```

Comment:
BOLD - FONTSIZE 128 ... to be honest that should be visible to the user regardless if he/she scroll down through this notes.

Reply:
Done. Moved to Section 'Disclaimer' in the beginning of the document.